src/main/java/org/gwtproject/event/shared/GwtEvent.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2009 Google Inc.
```

Comment:
Needs to be updated to “The GWT Project Authors”

Reply:
Done